src/master-playlist-controller.js

```diff
@@ -1417,6 +1448,8 @@ export class MasterPlaylistController extends videojs.EventTarget {
     this.mainSegmentLoader_.mimeType(mimeTypes[0]);
     if (mimeTypes[1]) {
       this.audioSegmentLoader_.mimeType(mimeTypes[1]);
+      // fired when video and audio is demuxed
+      this.tech_.trigger({type: 'usage', name: 'hls-demuxed'});
```

Comment:
We want this event to mean that video and audio is demuxed by _default_ instead of when alternate audio is present. However, it would still be nice to know when alternate audio is present. I suggest changing this event to `hls-alternate-audio`.

For the `hls-demuxed` event, I think a good place to put it is within `fillAudioTracks_`. Since the `DEFAULT` attribute for media tags is optional, the best way we can detect that video and audio are demuxed by default is if all audio renditions have a `URI`, since the lack of uri means the audio is muxed with the video for that rendition. `fillAudioTracks_` already has to loop through all the audio groups, so I suggest checking each one within the loop for a uri, and after the loop if they all had it, fire the event